src/style.css

```diff
@@ -0,0 +1,135 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
```

Comment:
Do not change the external indents for the block, you can create a class for body